lib/x11test.pm

```diff
@@ -251,10 +252,9 @@ sub check_new_mail_evolution {
     # Delete the message and expunge the deleted item if not used POP3
     if ($protocol != "POP") {
         send_key "ctrl-e";
-        if (check_screen "evolution_mail-expunge") {
-            send_key "alt-e";
-        }
-        assert_screen "evolution_mail-ready";
+        assert_screen [qw(evolution_mail-ready evolution_mail-expunge)];
```

Comment:
I'm not sure if we can change such code always, as there are scenarios where wrong match will happen first. E.g. in case of pop-ups or there is sequence prescribed (in such scenarios openQA sometimes matches second tag, even when first tag matches as well). So I would like to see verification runs for those.